config/routes.rb

```diff
@@ -105,6 +105,11 @@
           get 'add_group'
           post 'use_another_assignment_groups'
           get 'manage'
+          get 'assign_scans'
+          get 'download'
+          get 'get_names'
+          get 'assign_student_and_next'
+          get 'next_grouping'
```

Comment:
It gets called at the loading in of the page when you don't need to assign a student. The ajax fills in the content of the first exam this way.